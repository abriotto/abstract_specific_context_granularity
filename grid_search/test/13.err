  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:01, 54.64it/s] 13%|########################5                                                                                                                                                                  | 13/99 [00:00<00:01, 56.20it/s] 24%|#############################################3                                                                                                                                             | 24/99 [00:00<00:01, 74.45it/s] 32%|############################################################4                                                                                                                              | 32/99 [00:00<00:01, 63.30it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:00<00:00, 67.71it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:00<00:00, 67.59it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:00<00:00, 66.23it/s] 67%|############################################################################################################################6                                                              | 66/99 [00:00<00:00, 73.34it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:01<00:00, 73.10it/s] 83%|##########################################################################################################################################################8                                | 82/99 [00:01<00:00, 67.34it/s] 90%|########################################################################################################################################################################1                  | 89/99 [00:01<00:00, 63.68it/s] 98%|#######################################################################################################################################################################################2   | 97/99 [00:01<00:00, 66.28it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 66.20it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:00<00:00, 81.72it/s] 76%|##############################################################################################################################################1                                            | 19/25 [00:00<00:00, 87.17it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 74.06it/s]
