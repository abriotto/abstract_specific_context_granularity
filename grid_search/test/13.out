# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7008358240127563, "acc": 0.4940676689147949, "length": 3.505223512649536, "mode": "train", "epoch": 1}
{"loss": 0.6947948932647705, "acc": 0.49094197154045105, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6946074366569519, "acc": 0.5177578330039978, "length": 2.234785318374634, "mode": "train", "epoch": 2}
{"loss": 0.6951894760131836, "acc": 0.4855073094367981, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6876311898231506, "acc": 0.5315523743629456, "length": 1.6648446321487427, "mode": "train", "epoch": 3}
{"loss": 0.6793427467346191, "acc": 0.5756340622901917, "length": 1.66847825050354, "mode": "test", "epoch": 3}
{"loss": 0.6630241870880127, "acc": 0.6126799583435059, "length": 2.333477735519409, "mode": "train", "epoch": 4}
{"loss": 0.6384071111679077, "acc": 0.6349260210990906, "length": 2.7771739959716797, "mode": "test", "epoch": 4}
{"loss": 0.6032335162162781, "acc": 0.6681206822395325, "length": 2.7941250801086426, "mode": "train", "epoch": 5}
{"loss": 0.5942916870117188, "acc": 0.7052386999130249, "length": 3.1032607555389404, "mode": "test", "epoch": 5}
{"loss": 0.5420337319374084, "acc": 0.7263423800468445, "length": 3.018693208694458, "mode": "train", "epoch": 6}
{"loss": 0.5589047074317932, "acc": 0.7182027697563171, "length": 3.16847825050354, "mode": "test", "epoch": 6}
{"loss": 0.5266008377075195, "acc": 0.7521287202835083, "length": 3.1766161918640137, "mode": "train", "epoch": 7}
{"loss": 0.5599573850631714, "acc": 0.7552083730697632, "length": 3.1521739959716797, "mode": "test", "epoch": 7}
{"loss": 0.5010614991188049, "acc": 0.7623148560523987, "length": 3.167292356491089, "mode": "train", "epoch": 8}
{"loss": 0.536736011505127, "acc": 0.7585106492042542, "length": 3.38043475151062, "mode": "test", "epoch": 8}
{"loss": 0.49010100960731506, "acc": 0.775205671787262, "length": 3.2107043266296387, "mode": "train", "epoch": 9}
{"loss": 0.5398499965667725, "acc": 0.7582937479019165, "length": 3.38043475151062, "mode": "test", "epoch": 9}
{"loss": 0.4818687438964844, "acc": 0.7843730449676514, "length": 3.2560625076293945, "mode": "train", "epoch": 10}
{"loss": 0.5285806655883789, "acc": 0.7776363492012024, "length": 3.38043475151062, "mode": "test", "epoch": 10}
{"loss": 0.46701404452323914, "acc": 0.7981656193733215, "length": 3.2577719688415527, "mode": "train", "epoch": 11}
{"loss": 0.516672670841217, "acc": 0.7805895805358887, "length": 3.41304349899292, "mode": "test", "epoch": 11}
{"loss": 0.4437119960784912, "acc": 0.8097880482673645, "length": 3.286562442779541, "mode": "train", "epoch": 12}
{"loss": 0.5055786371231079, "acc": 0.7941293120384216, "length": 3.3967392444610596, "mode": "test", "epoch": 12}
{"loss": 0.43580102920532227, "acc": 0.8096268773078918, "length": 3.2902631759643555, "mode": "train", "epoch": 13}
{"loss": 0.5247611403465271, "acc": 0.7876850366592407, "length": 3.38043475151062, "mode": "test", "epoch": 13}
{"loss": 0.44436922669410706, "acc": 0.8140491843223572, "length": 3.3041577339172363, "mode": "train", "epoch": 14}
{"loss": 0.5275077819824219, "acc": 0.7871188521385193, "length": 3.38043475151062, "mode": "test", "epoch": 14}
{"loss": 0.4379827678203583, "acc": 0.8154410719871521, "length": 3.3041293621063232, "mode": "train", "epoch": 15}
{"loss": 0.528838574886322, "acc": 0.7648324370384216, "length": 3.3967392444610596, "mode": "test", "epoch": 15}
{"loss": 0.4266233742237091, "acc": 0.8133519887924194, "length": 3.334486722946167, "mode": "train", "epoch": 16}
{"loss": 0.4877239167690277, "acc": 0.7971581220626831, "length": 3.3967392444610596, "mode": "test", "epoch": 16}
{"loss": 0.4084961712360382, "acc": 0.8129678964614868, "length": 3.323702335357666, "mode": "train", "epoch": 17}
{"loss": 0.49126386642456055, "acc": 0.7993659973144531, "length": 3.3967392444610596, "mode": "test", "epoch": 17}
{"loss": 0.40840157866477966, "acc": 0.8159212470054626, "length": 3.3182246685028076, "mode": "train", "epoch": 18}
{"loss": 0.49958324432373047, "acc": 0.7883549332618713, "length": 3.5271739959716797, "mode": "test", "epoch": 18}
{"loss": 0.39781156182289124, "acc": 0.8230679631233215, "length": 3.356431007385254, "mode": "train", "epoch": 19}
{"loss": 0.46978840231895447, "acc": 0.7979507446289062, "length": 3.49456524848938, "mode": "test", "epoch": 19}
{"loss": 0.38709673285484314, "acc": 0.8279364705085754, "length": 3.3423376083374023, "mode": "train", "epoch": 20}
{"loss": 0.470516175031662, "acc": 0.7821557521820068, "length": 3.46195650100708, "mode": "test", "epoch": 20}
{"loss": 0.3930671811103821, "acc": 0.8310366272926331, "length": 3.3646559715270996, "mode": "train", "epoch": 21}
{"loss": 0.462942898273468, "acc": 0.7922044992446899, "length": 3.46195650100708, "mode": "test", "epoch": 21}
{"loss": 0.39049583673477173, "acc": 0.8349251747131348, "length": 3.3683712482452393, "mode": "train", "epoch": 22}
{"loss": 0.471760630607605, "acc": 0.7961956262588501, "length": 3.49456524848938, "mode": "test", "epoch": 22}
{"loss": 0.3771977126598358, "acc": 0.8399659395217896, "length": 3.4343276023864746, "mode": "train", "epoch": 23}
{"loss": 0.4642084836959839, "acc": 0.8029136657714844, "length": 3.6413042545318604, "mode": "test", "epoch": 23}
{"loss": 0.3768288791179657, "acc": 0.84542316198349, "length": 3.501244068145752, "mode": "train", "epoch": 24}
{"loss": 0.45823022723197937, "acc": 0.8018946647644043, "length": 3.7228260040283203, "mode": "test", "epoch": 24}
{"loss": 0.3741890788078308, "acc": 0.8475947976112366, "length": 3.516963481903076, "mode": "train", "epoch": 25}
{"loss": 0.46123868227005005, "acc": 0.8092448115348816, "length": 3.83695650100708, "mode": "test", "epoch": 25}
{"loss": 0.36108842492103577, "acc": 0.8426218628883362, "length": 3.546370506286621, "mode": "train", "epoch": 26}
{"loss": 0.45832717418670654, "acc": 0.8062633275985718, "length": 3.5597825050354004, "mode": "test", "epoch": 26}
{"loss": 0.3486640155315399, "acc": 0.8519220948219299, "length": 3.5037996768951416, "mode": "train", "epoch": 27}
{"loss": 0.45842522382736206, "acc": 0.8063953518867493, "length": 3.5923912525177, "mode": "test", "epoch": 27}
{"loss": 0.3376050591468811, "acc": 0.8585309982299805, "length": 3.464855670928955, "mode": "train", "epoch": 28}
{"loss": 0.45261526107788086, "acc": 0.806980311870575, "length": 3.57608699798584, "mode": "test", "epoch": 28}
{"loss": 0.3371336758136749, "acc": 0.8595836162567139, "length": 3.520444631576538, "mode": "train", "epoch": 29}
{"loss": 0.45562681555747986, "acc": 0.8165571689605713, "length": 3.7391304969787598, "mode": "test", "epoch": 29}
{"loss": 0.3327914774417877, "acc": 0.8652859330177307, "length": 3.4914345741271973, "mode": "train", "epoch": 30}
{"loss": 0.4473108649253845, "acc": 0.8196991682052612, "length": 3.70652174949646, "mode": "test", "epoch": 30}
{"loss": 0.3198910057544708, "acc": 0.8627153038978577, "length": 3.5343103408813477, "mode": "train", "epoch": 31}
{"loss": 0.43866467475891113, "acc": 0.8058009147644043, "length": 3.6086957454681396, "mode": "test", "epoch": 31}
{"loss": 0.34197351336479187, "acc": 0.8659282326698303, "length": 3.5018579959869385, "mode": "train", "epoch": 32}
{"loss": 0.4208972454071045, "acc": 0.8318896889686584, "length": 3.78804349899292, "mode": "test", "epoch": 32}
{"loss": 0.36181631684303284, "acc": 0.8476734757423401, "length": 3.608133554458618, "mode": "train", "epoch": 33}
{"loss": 0.43355196714401245, "acc": 0.8058196902275085, "length": 3.67391300201416, "mode": "test", "epoch": 33}
{"loss": 0.337380588054657, "acc": 0.8518251776695251, "length": 3.4780943393707275, "mode": "train", "epoch": 34}
{"loss": 0.42135748267173767, "acc": 0.8236997723579407, "length": 3.5923912525177, "mode": "test", "epoch": 34}
{"loss": 0.32155266404151917, "acc": 0.8653306365013123, "length": 3.5786218643188477, "mode": "train", "epoch": 35}
{"loss": 0.4023195505142212, "acc": 0.8303329348564148, "length": 3.83695650100708, "mode": "test", "epoch": 35}
{"loss": 0.29401296377182007, "acc": 0.8736554384231567, "length": 3.688042163848877, "mode": "train", "epoch": 36}
{"loss": 0.39894944429397583, "acc": 0.8346638083457947, "length": 3.9347825050354004, "mode": "test", "epoch": 36}
{"loss": 0.3035126328468323, "acc": 0.8800014853477478, "length": 3.902663230895996, "mode": "train", "epoch": 37}
{"loss": 0.4032360017299652, "acc": 0.8297101855278015, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.28239062428474426, "acc": 0.8870057463645935, "length": 3.9910905361175537, "mode": "train", "epoch": 38}
{"loss": 0.41388171911239624, "acc": 0.83152174949646, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.2989906370639801, "acc": 0.8848999738693237, "length": 3.9939656257629395, "mode": "train", "epoch": 39}
{"loss": 0.38654661178588867, "acc": 0.83423912525177, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.2672688663005829, "acc": 0.8904241919517517, "length": 3.995403289794922, "mode": "train", "epoch": 40}
{"loss": 0.4283272624015808, "acc": 0.8306159377098083, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.2791651487350464, "acc": 0.8880200982093811, "length": 3.9966835975646973, "mode": "train", "epoch": 41}
{"loss": 0.40702518820762634, "acc": 0.8288043737411499, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.26540035009384155, "acc": 0.8880466818809509, "length": 3.9983115196228027, "mode": "train", "epoch": 42}
{"loss": 0.4239153265953064, "acc": 0.8188406229019165, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.26326075196266174, "acc": 0.8996102213859558, "length": 3.9984991550445557, "mode": "train", "epoch": 43}
{"loss": 0.40249139070510864, "acc": 0.8242753744125366, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.292426735162735, "acc": 0.8929345607757568, "length": 3.99904203414917, "mode": "train", "epoch": 44}
{"loss": 0.3961157202720642, "acc": 0.8288043737411499, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.26594141125679016, "acc": 0.8828771114349365, "length": 3.9990439414978027, "mode": "train", "epoch": 45}
{"loss": 0.42856845259666443, "acc": 0.8097826838493347, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.2590349614620209, "acc": 0.8917129039764404, "length": 3.999418258666992, "mode": "train", "epoch": 46}
{"loss": 0.3893547058105469, "acc": 0.8469204306602478, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.24711677432060242, "acc": 0.8947102427482605, "length": 3.9962563514709473, "mode": "train", "epoch": 47}
{"loss": 0.39353451132774353, "acc": 0.8360508680343628, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.2517067492008209, "acc": 0.8980894088745117, "length": 3.9993131160736084, "mode": "train", "epoch": 48}
{"loss": 0.3999484181404114, "acc": 0.8251813054084778, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.2485860139131546, "acc": 0.9006628394126892, "length": 3.9957284927368164, "mode": "train", "epoch": 49}
{"loss": 0.4214187264442444, "acc": 0.8224637508392334, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.2502330243587494, "acc": 0.8953379392623901, "length": 3.9956963062286377, "mode": "train", "epoch": 50}
{"loss": 0.40882882475852966, "acc": 0.8378623723983765, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.24075442552566528, "acc": 0.8978594541549683, "length": 3.9899840354919434, "mode": "train", "epoch": 51}
{"loss": 0.42808470129966736, "acc": 0.8297100067138672, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.2591734528541565, "acc": 0.8972010016441345, "length": 3.9999420642852783, "mode": "train", "epoch": 52}
{"loss": 0.42842257022857666, "acc": 0.845108687877655, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.24251031875610352, "acc": 0.8998643159866333, "length": 3.997150421142578, "mode": "train", "epoch": 53}
{"loss": 0.4356931447982788, "acc": 0.8342391848564148, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.24960358440876007, "acc": 0.9017050862312317, "length": 3.9978842735290527, "mode": "train", "epoch": 54}
{"loss": 0.40791451930999756, "acc": 0.8260870575904846, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.2537691295146942, "acc": 0.8961415886878967, "length": 3.9954659938812256, "mode": "train", "epoch": 55}
{"loss": 0.41256386041641235, "acc": 0.8414856195449829, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.2322075068950653, "acc": 0.9008496403694153, "length": 3.9999184608459473, "mode": "train", "epoch": 56}
{"loss": 0.42943352460861206, "acc": 0.8269928097724915, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.24506203830242157, "acc": 0.9041964411735535, "length": 3.9999778270721436, "mode": "train", "epoch": 57}
{"loss": 0.41550713777542114, "acc": 0.8342391848564148, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2614421844482422, "acc": 0.9011551141738892, "length": 3.9999759197235107, "mode": "train", "epoch": 58}
{"loss": 0.4113636016845703, "acc": 0.8396739959716797, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.2372346669435501, "acc": 0.9057171940803528, "length": 3.999976873397827, "mode": "train", "epoch": 59}
{"loss": 0.40430599451065063, "acc": 0.8405795693397522, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.2394673377275467, "acc": 0.9054077863693237, "length": 3.999803066253662, "mode": "train", "epoch": 60}
{"loss": 0.4041563868522644, "acc": 0.8414856195449829, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.23441870510578156, "acc": 0.9077717065811157, "length": 3.9987926483154297, "mode": "train", "epoch": 61}
{"loss": 0.4005606174468994, "acc": 0.8442030549049377, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.2355901449918747, "acc": 0.906263530254364, "length": 3.9980459213256836, "mode": "train", "epoch": 62}
{"loss": 0.39917635917663574, "acc": 0.8523551821708679, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.24281175434589386, "acc": 0.9090607166290283, "length": 3.9999327659606934, "mode": "train", "epoch": 63}
{"loss": 0.4101335406303406, "acc": 0.8260870575904846, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.22126667201519012, "acc": 0.907845675945282, "length": 3.9999611377716064, "mode": "train", "epoch": 64}
{"loss": 0.40231209993362427, "acc": 0.8387682437896729, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.2344471663236618, "acc": 0.9060137271881104, "length": 3.9997496604919434, "mode": "train", "epoch": 65}
{"loss": 0.41521525382995605, "acc": 0.8514493703842163, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.22519929707050323, "acc": 0.9093665480613708, "length": 3.999976873397827, "mode": "train", "epoch": 66}
{"loss": 0.41242000460624695, "acc": 0.8206522464752197, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.22365668416023254, "acc": 0.9136138558387756, "length": 3.999171257019043, "mode": "train", "epoch": 67}
{"loss": 0.4044814705848694, "acc": 0.8460144996643066, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.2324724793434143, "acc": 0.9096698760986328, "length": 3.9999420642852783, "mode": "train", "epoch": 68}
{"loss": 0.42427048087120056, "acc": 0.8432971835136414, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.23613354563713074, "acc": 0.9054058790206909, "length": 3.9998276233673096, "mode": "train", "epoch": 69}
{"loss": 0.429715096950531, "acc": 0.8351448178291321, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.21636973321437836, "acc": 0.916361927986145, "length": 3.9999759197235107, "mode": "train", "epoch": 70}
{"loss": 0.41983360052108765, "acc": 0.8514491319656372, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.2108491063117981, "acc": 0.9160579442977905, "length": 3.9999866485595703, "mode": "train", "epoch": 71}
{"loss": 0.4363773465156555, "acc": 0.8423913717269897, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.221437007188797, "acc": 0.9137300848960876, "length": 3.994690418243408, "mode": "train", "epoch": 72}
{"loss": 0.42203307151794434, "acc": 0.8423913717269897, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.21355578303337097, "acc": 0.9124062061309814, "length": 3.9998939037323, "mode": "train", "epoch": 73}
{"loss": 0.41760462522506714, "acc": 0.85597825050354, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.20348350703716278, "acc": 0.9178820848464966, "length": 3.9999732971191406, "mode": "train", "epoch": 74}
{"loss": 0.403569757938385, "acc": 0.8686593770980835, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.20775206387043, "acc": 0.916968584060669, "length": 3.9999215602874756, "mode": "train", "epoch": 75}
{"loss": 0.417656272649765, "acc": 0.8387680649757385, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.2118123471736908, "acc": 0.9190416932106018, "length": 3.9984798431396484, "mode": "train", "epoch": 76}
{"loss": 0.42068490386009216, "acc": 0.8505435585975647, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.20066618919372559, "acc": 0.9200121164321899, "length": 3.999993324279785, "mode": "train", "epoch": 77}
{"loss": 0.4256891906261444, "acc": 0.8605073690414429, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.20634964108467102, "acc": 0.9215256571769714, "length": 3.9998321533203125, "mode": "train", "epoch": 78}
{"loss": 0.4240564703941345, "acc": 0.8650363683700562, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.22790329158306122, "acc": 0.910279393196106, "length": 3.9999847412109375, "mode": "train", "epoch": 79}
{"loss": 0.40676945447921753, "acc": 0.8514493107795715, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.22075064480304718, "acc": 0.9121029376983643, "length": 3.9994521141052246, "mode": "train", "epoch": 80}
{"loss": 0.43573594093322754, "acc": 0.8460144996643066, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.21314023435115814, "acc": 0.9166666269302368, "length": 3.9999983310699463, "mode": "train", "epoch": 81}
{"loss": 0.433743417263031, "acc": 0.8523551821708679, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.2010020762681961, "acc": 0.9178830981254578, "length": 3.9999969005584717, "mode": "train", "epoch": 82}
{"loss": 0.4110458791255951, "acc": 0.85597825050354, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.19676215946674347, "acc": 0.9172738194465637, "length": 3.999978542327881, "mode": "train", "epoch": 83}
{"loss": 0.4065278172492981, "acc": 0.8559783697128296, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.20495934784412384, "acc": 0.9218366146087646, "length": 3.9999897480010986, "mode": "train", "epoch": 84}
{"loss": 0.4078962206840515, "acc": 0.8550726175308228, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.18789136409759521, "acc": 0.925166130065918, "length": 3.999770164489746, "mode": "train", "epoch": 85}
{"loss": 0.4005986452102661, "acc": 0.8605071902275085, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.19890949130058289, "acc": 0.9242693781852722, "length": 3.9999852180480957, "mode": "train", "epoch": 86}
{"loss": 0.44180387258529663, "acc": 0.8586956262588501, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.20041567087173462, "acc": 0.9218270182609558, "length": 3.9996323585510254, "mode": "train", "epoch": 87}
{"loss": 0.4243522584438324, "acc": 0.8559783697128296, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.20612721145153046, "acc": 0.9194038510322571, "length": 3.9999947547912598, "mode": "train", "epoch": 88}
{"loss": 0.41726359724998474, "acc": 0.85597825050354, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.1867712289094925, "acc": 0.9251822829246521, "length": 3.9999923706054688, "mode": "train", "epoch": 89}
{"loss": 0.42667311429977417, "acc": 0.864130437374115, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.19518843293190002, "acc": 0.9207029938697815, "length": 3.994472026824951, "mode": "train", "epoch": 90}
{"loss": 0.46053606271743774, "acc": 0.85597825050354, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.2010185867547989, "acc": 0.9221407771110535, "length": 3.9999897480010986, "mode": "train", "epoch": 91}
{"loss": 0.4081598222255707, "acc": 0.8677536249160767, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.20022909343242645, "acc": 0.9259619116783142, "length": 3.9976043701171875, "mode": "train", "epoch": 92}
{"loss": 0.42718571424484253, "acc": 0.8523550629615784, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.19363641738891602, "acc": 0.9184901714324951, "length": 3.9999630451202393, "mode": "train", "epoch": 93}
{"loss": 0.4347318410873413, "acc": 0.8532609343528748, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.2024310827255249, "acc": 0.9260944724082947, "length": 3.9999871253967285, "mode": "train", "epoch": 94}
{"loss": 0.40435194969177246, "acc": 0.8614130616188049, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.18161723017692566, "acc": 0.9336823225021362, "length": 3.9998021125793457, "mode": "train", "epoch": 95}
{"loss": 0.42773279547691345, "acc": 0.8577899932861328, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.18415415287017822, "acc": 0.9282235503196716, "length": 3.9999959468841553, "mode": "train", "epoch": 96}
{"loss": 0.4249574542045593, "acc": 0.8614131212234497, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.17133088409900665, "acc": 0.9294403791427612, "length": 3.9999990463256836, "mode": "train", "epoch": 97}
{"loss": 0.42020684480667114, "acc": 0.8505434989929199, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.19344036281108856, "acc": 0.9236617684364319, "length": 3.999999523162842, "mode": "train", "epoch": 98}
{"loss": 0.44292256236076355, "acc": 0.8387680649757385, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.20674407482147217, "acc": 0.9184903502464294, "length": 3.9999477863311768, "mode": "train", "epoch": 99}
{"loss": 0.426532119512558, "acc": 0.8442030549049377, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.20686379075050354, "acc": 0.9215322732925415, "length": 3.999964475631714, "mode": "train", "epoch": 100}
{"loss": 0.4165785014629364, "acc": 0.8496378064155579, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7311509251594543
